src/app_service/service/message/service.nim

```diff
@@ -683,7 +683,7 @@ proc deleteMessage*(self: Service, messageId: string) =
 
 proc editMessage*(self: Service, messageId: string, msg: string) =
   try:
-    let allKnownContacts = self.contactService.getContacts()
+    let allKnownContacts = self.contactService.getContactsByGroup()
```

Comment:
Could we specify the `AllKnownContacts` group in the param here, or create a new `getAllContacts` function in the service that calls `getContactsByGroup(ContactsGroup.AllKnownContacts)`
Otherwise, it's a bit weird to call `ByGroup` with no group